blockchain/system/multicall.go

```diff
@@ -75,11 +75,7 @@ func (caller *ContractCallerForMultiCall) CallContract(ctx context.Context, call
 }
 
 // NewMultiCallContractCaller creates a new instance of ContractCaller for MultiCall contract.
```

Comment:
Please add comment that informs `state` must be temporal (copied) stateDB only used for multicall injection and call.

Reply:
Fixed in 4e065d14 by always running state.Copy()